test/e2e/tests/risingwave/manifests/storages/object-azure-blob.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Secret
+metadata: 
+  name: risingwave-azure-blob-credentials
+  namespace: ${E2E_NAMESPACE}
+stringData:
+  AccountName: ${E2E_AZURE_ACCOUNTNAME}
+  AccountKey: ${E2E_AZURE_ACCOUNTKEY}
+---
+apiVersion: risingwave.risingwavelabs.com/v1alpha1
+kind: RisingWave
+metadata:
+  name: ${E2E_RISINGWAVE_NAME}
+  namespace: ${E2E_NAMESPACE}
+spec:
+  storages:
+    meta:
+      memory: true
+    object:
+      azureBlob: 
+        secret:  risingwave-azure-blob-credentials
+        container: ${E2E_AZURE_CONTAINER}
+        root: risingwave
+        endpoint: ${E2E_AZURE_ENDPOINT}
+  global:
+    image: ghcr.io/risingwavelabs/risingwave:git-5cd14918233990d28b48a153fa8d11115089c148
```

Comment:
Same with the previous reason.

Reply:
I think we need to load it before the test case runs.

Reply:
Because the test case will run only when the environmental variable exists. I think the object_azure_blob test case will not automatically launch by pipeline. When I  run the test case by myself, I think it will automatically try to pull down the image.